Gruntfile.js

```diff
@@ -4,7 +4,8 @@ module.exports = function(grunt) {
         '**/*.js',
         '!node_modules/**',
         '!tests/node_modules/**',
-        '!selenium/**'
+        '!selenium/**',
+        '!tests/nightwatch.js'
```

Comment:
Is there a particular reason why this file needs to be excluded from linting and not just fixed?